notebooks/spark/AerospikeSparkPython.ipynb

```diff
@@ -5,7 +5,7 @@
    "metadata": {},
    "source": [
     "# Aerospike Connect for Spark Tutorial for Python\n",
-    "#### Tested with Spark connector 3.4.1, ASDB EE 6.0.0.0 EE, Java 8, Apache Spark 3.1.2, Python 3.7  and Scala 2.12.11 and [Spylon]( https://pypi.org/project/spylon-kernel/)"
+    "#### Tested with Spark connector 4.3.0, ASDB EE 6.4 EE, Java 8, Apache Spark 3.4.0, Python 3.10  and Scala 2.12.11"
```

Comment:
Scala 2.12.11 or 2.13?

Reply:
Removed Scala mention in the notebook.  I have spark 3.4.0 installed locally, which was built with Scala 2.12. I'm aware that spark version used in docker is Scala 2.13.